packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -47,7 +47,7 @@ type TApplicationShellProps = {
   onMenuItemClick?: (event: SyntheticEvent<HTMLAnchorElement>) => void;
   disableRoutePermissionCheck?: boolean;
   enableReactStrictMode?: boolean;
-  render?: () => JSX.Element;
+  render?: () => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.